src/main/java/tds/exam/results/mappers/TestMapper.java

```diff
@@ -38,4 +51,53 @@ private static String parseBankKeyFromId(final String itemId) {
 
         return itemId.split("-")[0];
     }
+
+    private static long parseLatestAcademicYear(final String academicYearString) {
+        // If this is a range date, pick the latest year
+        if (academicYearString.contains("-")) {
+            return Long.parseLong(academicYearString.split("-")[1]);
+        }
+
+        return Long.parseLong(academicYearString);
+    }
+
+    private static String createGradeStringFromGrades(final List<String> grades) {
```

Comment:
Normally I wouldn't ask this but could you add a small JavaDoc explaining what the purpose of this method is?  Following the code I see that if it is a single grade we return "Third", but there are multiples and not high school we return something, then finally handle high school ranges. 

I think it is one off enough that a bit of java doc would help future maintainers